### PR TITLE
Testing pybluez dependency for ble pairing

### PR DESCRIPTION
**Pull request by ShanavazJs**

### CODE REVIEW

File: bot_python_sdk/bluetooth_service.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from bluetooth.ble import DiscoveryService, BeaconService
+from bot_python_sdk.logger import Logger
+
+LOCATION = 'Bluetooth Service'
+
+
+
+class BluetoothService:
+
+    def discover_bluetooth_test(self):
```
Comment (pimstolk) [inline bot_python_sdk/bluetooth_service.py]: The SDK shouldn't discover devices.

File: bot_python_sdk/bluetooth_service.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            for addr, name in nearby_devices:
+                print("  %s - %s" % (addr, name))
+            discover_response = {"message" : "success"}
+        except Exception as exception:
+            Logger.error(LOCATION, 'Failed discover_bluetooth_test')
+            discover_response = {"message" : "failed"}
+            raise exception        
+        return discover_response
+
+    def discover_ble_test(self):
```
Comment (pimstolk) [inline bot_python_sdk/bluetooth_service.py]: The SDK shouldn't discover devices.

File: bot_python_sdk/bluetooth_service.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        except Exception as exception:
+            Logger.error(LOCATION, 'Failed discover_ble_test')
+            discover_response = {"message" : "failed"}
+            raise exception
+        return discover_response
+
+
+    def advertising_ble_test(self):
+        try:
+            service = BeaconService()
```
Comment (pimstolk) [inline bot_python_sdk/bluetooth_service.py]: I am not sure that BeaconService is the right thing to do.
You can't connect to it and exchange Characteristic to the mobile devices 

Please see:
https://github.com/BankingofThings/BoT-NodeJS-SDK/blob/master/lib/bleno/bleno-service.js

File: bot_python_sdk/bluetooth_service.py (hunk 1 above)
Comment (ShanavazJs) [reply bot_python_sdk/bluetooth_service.py]: I have removed the function.

File: bot_python_sdk/bluetooth_service.py (hunk 2 above)
Comment (ShanavazJs) [reply bot_python_sdk/bluetooth_service.py]: I have removed the method in latest commit.

File: bot_python_sdk/bluetooth_service.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+from bot_python_sdk.logger import Logger
+import bluetooth
+from bluetooth.ble import DiscoveryService, BeaconService
```
Comment (pimstolk) [inline bot_python_sdk/bluetooth_service.py]: DiscoveryService can go from the imports

**Pull request closed without merging**